serving/docker/dockerd-entrypoint-with-cuda-compat.sh

```diff
@@ -71,6 +80,9 @@ if [[ "$1" = "serve" ]]; then
 elif [[ "$1" = "partition" ]] || [[ "$1" = "train" ]]; then
     shift 1
     /usr/bin/python3 /opt/djl/partition/partition.py "$@"
+elif [[ "$1" = "quantize" ]]; then
```

Comment:
why did we introduce this? Could we do everything in partition?

Reply:
Sorry, this is hanging from previous implementation. Will remove.